shared/src/libs/enums/exception-message.enum.ts

```diff
@@ -4,6 +4,8 @@ const ExceptionMessage = {
   USER_NOT_FOUND: 'User with these credentials was not found.',
   INCORRECT_CREDENTIALS: 'Incorrect credentials.',
   USER_ALREADY_EXISTS: 'User already exists.',
+  INCORRECT_FILE_TYPE: 'Content type of the file is not in png, jpeg.',
```

Comment:
I think it could be improved a bit for better clarity:
`INCORRECT_FILE_TYPE: 'Content type of the file is not in PNG or JPEG',` or
`INCORRECT_FILE_TYPE: 'Content type of the file is not in PNG/JPEG',`